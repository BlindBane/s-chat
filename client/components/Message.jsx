import React from 'react';

function Message(props) {
  return (
    <div>
      <span>[{props.channel}] </span>
      <span>{props.username} says: </span>
      <span>{props.messageText} </span>
    </div>
  );
}

export default Message;
