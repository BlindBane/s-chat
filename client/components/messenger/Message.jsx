import React from 'react'
import { Link } from 'react-router-dom'

function Message (props) {
  return (
    <div>
      <Link to={`/message/${props.msgId}`}>
        <span>[{props.channel}] </span>
        <span>{props.username} says: </span>
        <span>{props.messageText} </span>
      </Link>
    </div>
  )
}

export default Message
