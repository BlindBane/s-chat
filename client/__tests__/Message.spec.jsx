import React from 'react'
import { shallow } from 'enzyme'
import Message from './../components/messenger/Message'

test('Message renders correctly', () => {
  const component = shallow(<Message />)
  expect(component).toMatchSnapshot()
})
