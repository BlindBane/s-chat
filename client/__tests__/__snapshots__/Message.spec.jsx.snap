// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders correctly 1`] = `
<div>
  <span>
    [
    ] 
  </span>
  <span>
     says: 
  </span>
  <span>
     
  </span>
</div>
`;
